setuptools/tests/integration/test_pip_install_sdist.py

```diff
@@ -202,13 +202,15 @@ def build_deps(package, sdist_file):
     "Manually" install them, since pip will not install build
     deps with `--no-build-isolation`.
     """
-    import tomli as toml
-
     # delay importing, since pytest discovery phase may hit this file from a
     # testenv without tomli
+    if sys.version_info >= (3, 11):
+        import tomllib
+    else:  # pragma: no cover
+        import tomli as tomllib
 
     archive = Archive(sdist_file)
-    info = toml.loads(_read_pyproject(archive))
+    info = tomllib.loads(_read_pyproject(archive))
```

Comment:
I think technically this is not currently needed to pass CI because our mypy configs cause this method to not be type-checked *yet*.

But it would be needed eventually or with #4192